smartmin/models.py

```diff
@@ -113,7 +113,7 @@ def unicode_csv_reader(utf8_data, dialect=csv.excel, **kwargs):
 
                         encoded.append(cell)
 
-                yield encoded
+                    yield encoded
```

Comment:
this was yielding the last row instead of the first one, so breaking the validation